app/actions/apiRequests.js

```diff
@@ -54,15 +55,24 @@ export const generateImage = () =>
         });
 
         cancelablePromise.then((response) => {
-            if (response.status < 200 || response.status >= 300) {
-                const error = new Error(response.statusText);
-                error.response = response;
-                throw error;
+            if (response.status >= 200 && response.status < 300) {
+                return response.blob()
+                    .then((blob) => {
+                        const date = moment().format("YYYY-MM-DD-HH:mm:ss");
+                        const worldFile = response.headers.get("World-File").replace(/\|/g, "\n");
+                        const worldFileBlob = new Blob([worldFile], { type: "text/plain" });
+                        saveAs(worldFileBlob, `map-${date}.pgw`);
```

Comment:
Do we want to prompt this every time? Should we instead add to the state and expose it with a button?